sdk/core/azure-core/inc/azure/core/internal/unique_handle.hpp

```diff
@@ -62,6 +62,5 @@ namespace Azure { namespace Core { namespace _internal {
   template <typename T> struct UniqueHandleHelper;
 
   // *** Now users can say UniqueHandle<T> if they want:
-  template <typename T, template <typename> class U = UniqueHandleHelper>
-  using UniqueHandle = typename U<T>::type;
+  template <typename T, typename U = UniqueHandleHelper<T>> using UniqueHandle = typename U::type;
```

Comment:
Should the `using ...` be on a new line?

Reply:
It is how clang-format formatted it. But if we really want the newline, we could put an empty line comment (`//`) after `>` and before `using`.

Reply:
I don't have an opinion one way or another.

This new approach is more readable/understandable than how it was written previously (as long as they are identical).

That said, both ways highlight a complexity in C++ templates that I have trouble parsing the syntax of and understanding.
Unrelated to this PR: For me, some source comments explaining how I should be reading this would be helpful.